pkg/agent/client/update.go

```diff
@@ -1,48 +1,8 @@
 package client
 
-import (
-	"bytes"
-	"fmt"
-
-	"github.com/spiffe/spire/proto/spire/api/node"
-	"github.com/spiffe/spire/proto/spire/common"
-)
+import "github.com/spiffe/spire/proto/spire/common"
 
 type Update struct {
 	Entries map[string]*common.RegistrationEntry
-	SVIDs   map[string]*node.X509SVID
 	Bundles map[string]*common.Bundle
 }
-
-func (u *Update) String() string {
```

Comment:
This function is not really used. The cache implementation already has rich logging of the changes to entries and such.